apps/web-giddh/src/app/multi-currency-reports/balance-sheet/components/balance-sheet-grid/components/balance-sheet-grid-row/balance-sheet-report-grid-row.component.html

```diff
@@ -0,0 +1,100 @@
+<div
+    class="row pl-grid-row"
+    [financial-accordion]="groupDetail"
+    *ngIf="groupDetail.groupName && (groupDetail.isVisible || groupDetail.isCreated)"
+    [ngClass]="{ 'isHidden': !groupDetail.isVisible }"
```

Comment:
```suggestion
    [ngClass]="{ 'd-none': !groupDetail.isVisible }"
```
remove this class if not in use

Reply:
Done